### PR TITLE
Adds activity groups to styleguide

### PR DESCRIPTION
**Pull request by floriank**
This adds activity groups to the living styleguide which aim to be a
general solution to the comment histories, journals and the activity
view.

Meets https://community.openproject.org/work_packages/19014

We should discuss if these should already replace the existing styles in the living styleguide for the work package details pane, as these are similar in style and design goal.

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://floriank-feature-19014-styleguide-activity-groups.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://floriank-feature-19014-styleguide-activity-groups.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by myabc** [comment]
@floriank Idea + structure looks good. I don't see any CSS rules in `_activities.sass` though. Shouldn't we wait for an implementation to merge?

File: app/assets/stylesheets/content/_activities.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        </div>
+        <div class="activity-group--entry">
+            <div class="activity-group--avatar">
+                <img alt="Avatar" class="avatar" src="https://s3.amazonaws.com/uifaces/faces/twitter/teeragit/128.jpg" title="OpenProject Admin">
+            </div>
+            <div class="activity-group--entry--content">
+                <h2 class="activity-group--entry--heading">
+                    Demoproject: lorem ipsum dolor sit amet
+                    <small>
+                        <span class="activity-group--entry--creator">anastasia of Astora</span>
```
Comment (myabc) [inline app/assets/stylesheets/content/_activities.md:39]: @floriank I would only use the `--` separator once per class name.

**Comment by floriank** [comment]
There is no real styling done for that as of now, i only prepared the DOM (and then got interrupted).

Also, currently, we have some explicit styling for the work package details pane activities, which I'd like to remove with introduction of an activity group component

**Comment by myabc** [comment]
@floriank –

> Also, currently, we have some explicit styling for the work package details pane activities, which I'd like to remove with introduction of an activity group component

I'm definitely in favour of this approach.

Do you want to provide the implementation?

**Comment by floriank** [comment]
If I had the time right now, I'd supose I could do it, but I am busy with deign bugs at the moment, relating to the Zurb Foundation epic.

I'd put this on hold for he moment - at least until I corrected the double namespacing in the DOM.

**Comment by NobodysNightmare** [comment]
@floriank Will we revive this PR after the 4.1 fuzz is over?

**Comment by floriank** [comment]
Yes, we will. I seriously want to make a component out of the activity sections - and all this needs is a small change to the naming convention in the CSS - as well as the transfer of the specific solution for the WP Details pane.

**Comment by floriank** [comment]
Branch is still with me, but i do not have the time to work on this at the moment.

**Pull request closed without merging**